### PR TITLE
Remember last visited variable or visualization

### PR DESCRIPTION
**Pull request by dmfalke**

### CODE REVIEW

File: src/lib/workspace/SessionPanel.tsx [hunk 1]
```diff
@@ -36,6 +37,18 @@ export function SessionPanel(props: Props) {
   const filteredEntities = uniq(
     sessionState.session?.filters.map((f) => f.entityId)
   );
+  const location = useLocation();
+  const [lastVarPath, setLastVarPath] = useState('');
+  const [lastVizPath, setLastVizPath] = useState('');
+  useEffect(() => {
+    console.log({ location });
```
Comment (jtlong3rd) [inline src/lib/workspace/SessionPanel.tsx:44]: Were you planning to delete this `console.log` before merging, or keep it around for now?

**Review by jtlong3rd — APPROVED** [review]
Looks good to me! I had one inline comment.

File: src/lib/workspace/SessionPanel.tsx (hunk 1 above)
Comment (dmfalke) [reply src/lib/workspace/SessionPanel.tsx:44]: I was planning to delete, thanks!

File: src/lib/workspace/SessionPanel.tsx [hunk 2]
```diff
@@ -57,13 +69,22 @@ export function SessionPanel(props: Props) {
               deleteSession={deleteSession}
             />
             <Route
-              path={[`${routeBase}/variables/:entityId?`, `${routeBase}`]}
-              render={(props: RouteComponentProps<{ entityId?: string }>) => (
+              path={[
+                `${routeBase}/variables/:entityId?/:variableId?`,
+                `${routeBase}`,
```
Comment (bobular) [inline src/lib/workspace/SessionPanel.tsx:74]: Second element could be just `routeBase` - but there is also some nice symmetry to the way it is written now.

**Review by bobular — APPROVED** [review]
Tested functionally - can't find any holes!

Much more elegant than the solution I was going to try ;-) 

It is a shame that the `history` from `useHistory()` isn't searchable though.

File: src/lib/workspace/SessionPanel.tsx (hunk 2 above)
Comment (dmfalke) [reply src/lib/workspace/SessionPanel.tsx:74]: I decided to just leave it. It's doesn't really hurt much :slightly_smiling_face:

**Pull request merged**